### PR TITLE
Updating copyright headers to match current best practices

### PR DESCRIPTION
**Pull request by tmichalak**
Made applicable changes mentioned in https://github.com/SymbiFlow/ideas/issues/47
* Renamed the `COPYING` file
* Removed `All rights reserved`
* Updated the year
* Added short version of ISC license header to all *.cc and *.h files
* Added short version of ISC license header to all *.py scripts
* Added short version of ISC license header to all bash scripts
* Added short version of ISC license header to all Makefiles
* Added  the SPDX identifier to the license headers

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2017  The Project X-Ray Authors. All rights reserved.
+Copyright (C) 2020  The Project X-Ray Authors.
```
Comment (mithro) [inline LICENSE:1]: `2017-2020`

File: lib/big_endian_span_test.cc [hunk 2]
```diff
@@ -1,3 +1,18 @@
+/*
```
Comment (mithro) [inline lib/big_endian_span_test.cc:1]: Let's use the short version of the ISC header.

Also add the SPDX identifier.

Comment (tmichalak) [reply lib/big_endian_span_test.cc:1]: @mithro How does a short version look like?

**Comment by tmichalak** [comment]
@mithro We also want a license headers in `.py` and `.tcl` files, right? Also with the short version of the ISC header?

**Comment by mithro** [comment]
@tmichalak - Any code file, `.py`, `.tcl`, `Makefile`, `.sh`, etc......

File: lib/big_endian_span_test.cc (hunk 2 above)
Comment (mithro) [reply lib/big_endian_span_test.cc:1]: See first version in https://github.com/SymbiFlow/prjxray/issues/1300

File: LICENSE (hunk 1 above)
Comment (tmichalak) [reply LICENSE]: Done

File: lib/big_endian_span_test.cc (hunk 2 above)
Comment (tmichalak) [reply lib/big_endian_span_test.cc:1]: Done

File: minitests/roi_harness/runme.sh [hunk 3]
```diff
@@ -1,3 +1,12 @@
+#!/bin/bash
+# SPDX-License-Identifier: ISC
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
 #!/usr/bin/env bash
```
Comment (aimylios) [inline minitests/roi_harness/runme.sh:10]: Duplicate shebang.

File: vagrant/profile.d/vivado.sh [hunk 4]
```diff
@@ -1,3 +1,12 @@
+#!/bin/bash
+# SPDX-License-Identifier: ISC
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
 #!/bin/sh
```
Comment (aimylios) [inline vagrant/profile.d/vivado.sh:10]: Duplicate shebang.

File: minitests/roi_harness/runme.sh (hunk 3 above)
Comment (tmichalak) [reply minitests/roi_harness/runme.sh]: Thanks for spotting this. Fixed.

File: vagrant/profile.d/vivado.sh (hunk 4 above)
Comment (tmichalak) [reply vagrant/profile.d/vivado.sh]: Thanks for spotting this. Fixed.

File: .github/kokoro/db-full.sh [hunk 5]
```diff
@@ -1,4 +1,13 @@
 #!/bin/bash
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline .github/kokoro/db-full.sh:2]: The SPDX codes go after the license, see the other repository.

Comment (tmichalak) [reply .github/kokoro/db-full.sh:2]: I thought I did exactly as I saw in the other repo...
Not the point though. Is such header the desired one?
<pre>
#!/bin/bash
#
# Copyright (C) 2017-2020  The Project X-Ray Authors.
#
# Use of this source code is governed by a ISC-style
# license that can be found in the LICENSE file or at
# https://opensource.org/licenses/ISC
#
# SPDX-License-Identifier: ISC
#
</pre>

Comment (tmichalak) [reply .github/kokoro/db-full.sh:2]: @mithro if the header I pasted above is correct, just let me know and will go on with updating them accordingly.

Comment (mithro) [reply .github/kokoro/db-full.sh:2]: Extra `#` at the end here.

File: download-latest-db.sh [hunk 6]
```diff
@@ -1,4 +1,18 @@
 #!/usr/bin/env bash
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline download-latest-db.sh:8]: Repeated license here.

Comment (tmichalak) [reply download-latest-db.sh:8]: Fixed.

**Comment by tmichalak** [comment]
@mithro I fixed the bug you mentioned and also added a license check to Travis. The checks currently fail with:
```
ERROR: fasm does not have the LICENSE file.
ERROR: python-sdf-timing does not have the LICENSE file.
```
which shall go away once the licensing PRs for fasm and python-sdf-timing are merged.

**Comment by mithro** [comment]
This fixed https://github.com/SymbiFlow/prjxray/issues/1306

**Pull request merged**